docs/DeveloperGuide.md

```diff
@@ -365,8 +387,175 @@ appropriate message through the `TextUi` class.
 
 ## Glossary
 
-* *glossary item* - Definition
+* *Instrument* - Represents assets that can be traded. Most common examples are stocks and foreign currency.
+* *Etf* - Known as Exchange Traded Funds, they are a type of instrument that tracks the performance of a particular asset. 
+* *Forex* - Foreign exchange market for trading currencies. An example included is the USDSGD exchange rate.
+* *Crypto* - Cryptocurrencies are digital currencies that is secured by cryptography methods.
```

Comment:
remove "Cryptocurrencies are"

Reply:
Rephrase to "Digital currencies that are secured by cryptography methods.